src/components/Table/DrawerListTable/index.vue

```diff
@@ -177,6 +176,9 @@ export default {
       }
       const resolvedRoute = this.resolveRoute(detailRoute)
       let title = cellValue || row.name
+      if (formatterArgs.getTitle) {
+        title = formatterArgs.getTitle({ col, row, cellValue })
+      }
       let resource = resolvedRoute?.meta?.title || resolvedRoute?.name
       resource = resource.replace('Detail', '').replace('详情', '')
 
```

Comment:
I cannot determine the specific differences between two codes without having access to them. Please share the code you would like me to compare.